openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -198,25 +198,36 @@
   narrative:
     - key: a
       text: |
-        Organizational review and analysis of information system
-        audit records for indications of organization-defined
-        inappropriate or unusual activity is outside the scope
-        of OpenShift configuration.
+        While OpenShift can generate audit data, and securely transport
+        the data to a SIEM, OpenShift does not provide automated mechanisms
+        (e.g. email, text messages) after system events. This functionality
+        would fall to third party software such as Splunk, Elastic, etc.
+        
+        To configure an IdP, see:
+        https://docs.openshift.com/container-platform/latest/authentication/identity_providers/configuring-ldap-identity-provider.html#configuring-ldap-identity-provider
```

Comment:
We could update the AU-6(a) response to further elaborate on what OpenShift can do, such as generate and transport audit data, if desired. No reason not to!

Portions of this change do not relate to the underlying control need rewording. Namely:
*  ``..OpenShift does not provide automated mechanisms after system events..``: AU-6(a) reflects organizational process (eg human review). Could you share how this sentence addresses the control?
* ``This functionality would fall to third party software....``: Since this is an organizational procedural control, this would not fall to third-party tools.
* ``To configure an IdP....``: Configuring OpenShift to external authentication does not seem relevant to this control. Could you share your logic for this?